kedro-telemetry/kedro_telemetry/plugin.py

```diff
@@ -30,6 +30,10 @@
 HEAP_APPID_PROD = "2388822444"
 HEAP_ENDPOINT = "https://heapanalytics.com/api/track"
 HEAP_HEADERS = {"Content-Type": "application/json"}
+KNOWN_CI_ENV_VAR_KEYS = [
```

Comment:
The PR mentioned list is faster to iterate. The speed here doesn't have any impact, but we usually defined these constant values with `tuple`.

see: https://github.com/kedro-org/kedro/blob/941440aa4a5815e8d8157d167ec581524a1f6e20/kedro/io/core.py#L31

Reply:
I'm sure this is a negligible micro optimization, but in any case +1 to turning this to a `tuple` 👍🏽

Reply:
(Although see below for a way of using a `set` instead)